gradle/libs.versions.toml

```diff
@@ -53,7 +53,7 @@ android-desugarJdkLibs = "com.android.tools:desugar_jdk_libs:1.0.10"
 
 # First-party libraries
 aps-sublimeFuzzy = "com.github.android-password-store:sublime-fuzzy:1.0.0"
-aps-zxingAndroidEmbedded = "com.github.android-password-store:zxing-android-embedded:4.1.0-aps"
+aps-zxingAndroidEmbedded = "com.github.android-password-store:zxing-android-embedded:4.2.0"
```

Comment:
The commit message says move to Maven Central, but this still seems to reference our fork.

Reply:
It does still reference our fork because we still maintain one, but the built binaries are now on [MavenCentral](https://oss.sonatype.org/service/local/repositories/releases/content/com/github/android-password-store/zxing-android-embedded/) rather than JitPack.